_posts/2023-11-20-securing-application-with-other-oidc.adoc

```diff
@@ -0,0 +1,274 @@
+```
+The application will now begin to build. This will take a couple of minutes.
+The build can be observed using:
+```
+oc get build -w
+```
+Once complete, you can follow the deployment of the application using:
+```
+oc get deployment oidc-app -w
+```
+Alternatively, you can check status directly from the OpenShift web console.
+
+=== Behind the scenes
+While our application is building, let’s take a closer look at our application.
+
+. Examine the https://github.com/PrarthonaPaul/elytron-examples/blob/simple-webapp-okta/simple-webapp-okta/pom.xml[pom.xml] file. Notice that it contains an openshift profile. A profile in Maven lets you create a set of configuration values to customize your application build for different environments. The openshift profile in this example defines a configuration that will be used by the WildFly Helm Chart when provisioning the WildFly server on OpenShift.
+
+```
+<profiles>
+        <profile>
+            <id>openshift</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.wildfly.plugins</groupId>
+                        <artifactId>wildfly-maven-plugin</artifactId>
+                        <version>${version.wildfly.maven.plugin}</version>
+                        <configuration>
+                            <feature-packs>
+                                <feature-pack>
+                                    <location>org.wildfly:wildfly-galleon-pack:${version.wildfly}</location>
+                                </feature-pack>
+                                <feature-pack>
+                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.wildfly.cloud.galleon.pack}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <layers>
+                                <layer>cloud-server</layer>
+                                <layer>elytron-oidc-client</layer>
+                            </layers>
+                            <filename>simple-webapp-okta.war</filename>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>package</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+```
+
+* *wildfly-maven-plugin* provisions a WildFly server with the specified layers with our application deployed.
+* *elytron-oidc-client* automatically adds the native OIDC client subsystem to our WildFly installation.
+
+. Examine the https://github.com/PrarthonaPaul/elytron-examples/blob/simple-webapp-okta/simple-webapp-okta/src/main/webapp/WEB-INF/web.xml[web.xml] file.
```

Comment:
Link will need to be updated (here and in other places) since it references your repo. Just adding a note so we don't forget.